dev/io.openliberty.jakarta.xmlBinding.4.0/bnd.bnd

```diff
@@ -31,12 +39,17 @@ DynamicImport-Package: \
   org.glassfish.jaxb.*
 
 Include-Resource:\
-  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;4.0.0;EXACT}!/!(META-INF/maven/*|module-info.class)
+  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;4.0.0;EXACT}!/!(META-INF/maven/*|module-info.class), \
+  io/openliberty/jakarta=${bin}/io/openliberty/jakarta
+  jakarta/xml/bind=${bin}/jakarta/xml/bind
+
+Multi-Release: true
 
-instrument.disabled: true
+instrument.ffdc: false
```

Comment:
I think you only want instrumentation for io.openliberty package.  If so you should have a filter.  In general I would like to not have anything added to this bundle, but you can see additional review comments about that.